src/Controls/src/Core/Platform/GestureManager/GesturePlatformManager.Android.cs

```diff
@@ -211,13 +218,53 @@ void SetupGestures()
 
 			// Always unsubscribe first to avoid duplicates
 			platformView.Touch -= OnPlatformViewTouched;
+			platformView.Focusable = _focusableDefaultValue;
 
 			if (shouldAddTouchEvent)
 			{
 				platformView.Touch += OnPlatformViewTouched;
+
+				// If we have a TapGestureRecognizer, we need to handle key presses
+				var tapRecognizers = View.GestureRecognizers.OfType<TapGestureRecognizer>().Where(x => x.NumberOfTapsRequired == 1);
+				if (tapRecognizers.Any())
+				{
+					_innerKeyListener = new InnerKeyListener(tapRecognizers, View);
+				}
+				platformView.SetOnKeyListener(_innerKeyListener);
```

Comment:
instead of keylistener here I think we just want to subscribe to the Key events we want

If we set the listener that's going to break anyone that's trying to subscribe to these events